account_payment_order/models/account_move.py

```diff
@@ -38,6 +39,12 @@ def _compute_payment_order_ok(self):
                 payment_mode = move.payment_mode_id
             move.payment_order_ok = payment_mode.payment_order_ok
 
+    def _compute_payment_line_count(self):
+        for move in self:
+            move.payment_line_count = self.env["account.payment.line"].search_count(
```

Comment:
Use `read_group` instead for better performance.

Reply:
this is an interesting one.
I didn't imagine that Odoo uses def search in stead of def _search in search_count.
Any idea why they do this ?

Reply:
Well, I was talking about doing something similar to:

https://github.com/OCA/bank-payment/blob/2f3def4c0306dc4dd1c0808383451dc344e4d94e/account_payment_order_grouped_output/models/account_payment_order.py#L21-L32

Reply:
In Odoo 17.0 the search_count performance issue has been fixed, cf.  https://github.com/odoo/odoo/blob/17.0/odoo/models.py#L1582